packages/auth/src/providers/cognito/utils/oauth/cancelOAuthFlow.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { OAuthStore } from '../types';
+
+import { createOAuthError } from './createOAuthError';
+
+import { handleFailure } from './index';
```

Comment:
Nit: For consistency with above and generally preferring directly accessing sibling functions
```suggestion
import { handleFailure } from './handleFailure';
```

Reply:
I already tried this import but it makes the unit test to fail.  It is not mocking it correctly

Reply:
Can you update the mock import?

Reply:
done